lib/screens/qr_code_screen.dart

```diff
@@ -0,0 +1,105 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:superformula_qr_code/models/seed_model.dart';
+import 'package:superformula_qr_code/services/api_service.dart';
+import 'package:superformula_qr_code/shared/app_routes.dart';
+import 'package:http/http.dart' as http;
+
+class QrCodeScreen extends StatefulWidget {
+  const QrCodeScreen({Key? key}) : super(key: key);
+
+  @override
+  State<QrCodeScreen> createState() => _QrCodeScreenState();
+}
+
+class _QrCodeScreenState extends State<QrCodeScreen> {
```

Comment:
**bug/question:** building on my prior comment, when I tapped "Get new code" I received the SocketException I was expecting the first time. 
![Simulator Screen Shot - iPhone 13 - 2021-11-23 at 14 27 12](https://user-images.githubusercontent.com/53348890/143090958-73f9d61c-b457-41f3-bcb6-d9877221fdae.png)

However, I still saw the timer. We shouldn't be displaying a timer right? We have no data to go off of and are assuming an expiration time. As a user, if something went wrong I'd expect to see that "get a new code" button, "try again" or "take my out of here" type of action.

Reply:
You're right, this was an oversight. We should only display the timer once we know we have data from the api.